ui/app/utils/openapi-to-attrs.js

```diff
@@ -0,0 +1,84 @@
+import DS from 'ember-data';
+const { attr } = DS;
+import { assign } from '@ember/polyfills';
+
+export const expandOpenApiProps = function(props) {
+  let attrs = {};
+  // expand all attributes
+  for (let prop in props) {
+    let details = props[prop];
+    if (details.deprecated === true) {
+      continue;
+    }
+    if (details.type === 'integer') {
+      details.type = 'number';
+    }
+    let editType = details.type;
+    if (details.format === 'seconds') {
```

Comment:
will seconds always mean it's a ttl?

Reply:
that was my understanding. @kalafut is this the case?

Reply:
correction: seconds means it's a duration, meaning a ttl form field would make most sense